sentences/ro/_common.yaml

```diff
@@ -1,12 +1,12 @@
 language: ro
 responses:
   errors:
-    no_intent: "Scuze, nu am înțeles cererea"
-    no_area: "Nu există nici o arie numită {{ area }}"
-    no_domain: "Aria {{ area }} nu conține {{ domain }}"
-    no_device_class: "Aria {{ area }} nu conține {{ device_class }}"
-    no_entity: "Niciun dispozitiv sau entitate cu numele {{ entity }}"
-    handle_error: "A apărut o eroare în timpul procesării cererii"
+    no_intent: "Îmi pare rău, nu am înțeles. Poți, te rog, să repeți?"
```

Comment:
Cred ca "cererea" e importanta in acest context, in care lipste fix intent-ul. E OK partea cu "poti sa repeti", dar cred ca e important sa prindem si "cererea"